### PR TITLE
Remove extra command and fix reference to profile

### PR DESCRIPTION
**Pull request by mtilson**
L348: The service is not being created in this section, looks like a copy/paste typo
L354: The profile file is `fine-grained.json`, not `fine-pod.json`

### CODE REVIEW

File: content/en/docs/tutorials/clusters/seccomp.md [hunk 1]
```diff
@@ -345,14 +345,13 @@ only the privileges they need.
 Clean up that Pod and Service before moving to the next section:
```
Comment (annajung) [inline content/en/docs/tutorials/clusters/seccomp.md:345]: Remove "Service" here as well

**Review by annajung — COMMENTED** [review]
Thank you for catching this!

I confirmed that service does not get created in the [Create Pod with seccomp profile that causes violation](https://github.com/kubernetes/website/blob/main/content/en/docs/tutorials/clusters/seccomp.md#create-pod-with-seccomp-profile-that-causes-violation) step

File: content/en/docs/tutorials/clusters/seccomp.md (hunk 1 above)
Comment (mtilson) [reply content/en/docs/tutorials/clusters/seccomp.md]: Done

**Review by pjbgf — APPROVED** [review]
nice catch, thank you for the amendments. :+1: 

/approve

**Comment by sftim** [comment]
@pjbgf did you mean /lgtm ?

**Comment by pjbgf** [comment]
@sftim yes, my bad.

/lgtm

**Pull request merged**